pinot-controller/src/main/resources/static/js/init.js

```diff
@@ -67,13 +67,13 @@ $(document).ready(function() {
     // execute query and draw the results
     var query = EDITOR.getValue().trim();
     var traceEnabled = document.getElementById('trace-enabled').checked;
-    var groupByModeSQL = document.getElementById('group-by-mode-sql').checked;
-    var responseFormatSQL = document.getElementById('response-format-sql').checked;
+    var groupByModePQL = document.getElementById('group-by-mode-pql').checked;
+    var responseFormatPQL = document.getElementById('response-format-pql').checked;
     var queryOptions = undefined;
-    if (groupByModeSQL === true) {
+    if (groupByModePQL !== true) {
       queryOptions = "groupByMode=sql";
     }
-    if (responseFormatSQL === true) {
+    if (responseFormatPQL !== true) {
       if (queryOptions === undefined) {
```

Comment:
Is the goal to invoke the new sql endpoint, or just set these options? If it's the former, we need a new endpoint on the controller "/sql", similar to "pql" which is in PqlQueryResource. Inside executeQuery method (called on line 83) we need to invoke "sql" endpoint accordingly.

Reply:
Provides a `/sql` endpoint now.